### ISSUE TITLE
Add ability to pan maps via the keyboard

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/earthworks#1146]
SODA suggestion - maps should be pannable using keyboard navigation (arrow keys)

**Comment by thatbudakguy** [comment]
we found in a meeting that leaflet has a `keyboard` setting and it is indeed possible to use the keyboard to pan when you have the entire map focused, like the one on this page: https://leafletjs.com/index.html#map-keyboard

**Comment by thatbudakguy** [comment]
(this might belong in GBL proper instead of earthworks)

**Comment by dnoneill** [comment]
@thatbudakguy I think its GBL and sul-embed.

**Comment by dnoneill** [comment]
Checked. Already working in sul-embed.

**Comment by dnoneill** [comment]
This is already working in geoblacklight as well. see https://earthworks-uat.stanford.edu/catalog/VAC3073-M-00794
